.github/workflows/bootzooka-ci.yml

```diff
@@ -49,10 +49,9 @@ jobs:
         working-directory: ./ui
 
   publish:
-    if: github.event_name != 'pull_request' && github.ref == 'refs/heads/master'
-    needs: [ verify ]
+    needs: [verify]
     runs-on: ubuntu-24.04
-
+    if: github.ref == 'refs/heads/master'
```

Comment:
you don't need the `github.event_name != 'pull_request'` check? so that this doesn't run on PRs to master?